src/coreclr/jit/gentree.h

```diff
@@ -5793,6 +5793,31 @@ enum RMWStatus
     STOREIND_RMW_INDIR_UNEQUAL     // Indir to read value is not equivalent to indir that writes the value
 };
 
+#ifdef DEBUG
+inline const char* RMWStatusString(RMWStatus status)
+{
+    switch (status)
+    {
+        case STOREIND_RMW_STATUS_UNKNOWN:
+            return "STATUS_UNKNOWN";
```

Comment:
Instead of having short abbreviation, just have easy to read reasons like `"Status Unknown", "Dst candidate is op1"`, etc.